packages/@okta/vuepress-site/code/react/okta_react/index.md

```diff
@@ -346,14 +351,12 @@ export default withOktaAuth(class SignIn extends Component {
 The component for this route (LoginCallback) comes with `@okta/okta-react`. It handles token parsing, token storage, and redirecting to a protected page if one triggered the sign in.
 
 ### Connect the Routes
-Update `src/App.jsx` to include your project components and routes. `Security` is the component that controls the authentication flows, so it requires your OpenId Connect configuration. By default, `@okta/okta-react` redirects to Okta's sign in page when the user isn't authenticated.
+Update `src/App.js` to include your project components and routes. `Security` is the component that controls the authentication flows, so it requires your OpenId Connect configuration. By default, `@okta/okta-react` redirects to Okta's sign in page when the user isn't authenticated.
 
 In this example, `onAuthRequired` is overridden to redirect to the custom sign-in route instead, which requires a component that is a descendent of Router to have access to `react-router`'s `history`.  Other router libraries will have their own methods of managing browser history:
 
-`src/App.jsx` and `src/AppWithRouterAccess.jsx` using function-based components:
+Update `src/App.js` to use function-based components:
```

Comment:
create-react-app is already using function-based components.  Here the guide is electing to having the dev-created components be function-based, so I don't really like "Update...to use"

Reply:
Should I keep the same words as before? The only thing I don't like about it is there are two code blocks with no words in between them.